projects/packages/analyzer/scripts/jetpack-slurper.php

```diff
@@ -30,8 +30,6 @@
 
 require dirname( dirname( __FILE__ ) ) . '/vendor/autoload.php';
```

Comment:
https://phabricator.wikimedia.org/T273574 strikes again.

```suggestion
require dirname( __DIR__ ) . '/vendor/autoload.php';
```